docs/multicluster/user-guide.md

```diff
@@ -398,11 +398,11 @@ supported.
 
 After the Gateway is created, Multi-cluster Controller will be responsible
 for exporting the cluster's network information to other member clusters
-through the leader cluster, including the cluster's Gateway IP and Service
-CIDR. Multi-cluster Controller will try to discover the cluster's Service CIDR
-automatically, but you can also manually specify the `serviceCIDR` option in
-ConfigMap `antrea-mc-controller-config`. In other member clusters, a
-ClusterInfoImport CR will be created for the cluster which includes the
+through the leader cluster, including the cluster's Gateway IP and Service CIDR.
+Before K8s v1.33.0, Multi-cluster Controller will try to discover the cluster's Service
```

Comment:
Consider:

If a member cluster is running a K8s version earlier than v1.33.0, the Multi-cluster Controller will attempt to automatically discover the cluster’s Service CIDR when the `serviceCIDR` field in the `antrea-mc-controller-config` ConfigMap is not set. For clusters with Kubernetes v1.33.0 or later, the `serviceCIDR` field is currently required and must be explicitly specified, until support for Service CIDR auto-discovery with newer Kubernetes versions is added.